corehq/apps/sms/models.py

```diff
@@ -1508,7 +1508,6 @@ class SQLMobileBackend(SyncSQLToCouchMixin, models.Model):
 
     class Meta:
         db_table = 'messaging_mobilebackend'
-        unique_together = ('domain', 'name')
```

Comment:
i realized that you can (soft) delete a backend and then create a new one with the same name (which is fine to do), but then this constraint fails.  so since the unique name per domain is enforced in the ui that should be enough for this